best-practices.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Best Practices
+description: Learn our best practices for building and maintaining your Chalk solution
+---
+
+# Best Practices
+
+With Chalk, the same solution can be implemented in a number of different ways. Below are some guidelines
+and recommended patterns for building and maintaining your Chalk solution!
+
+## Data Sources/Integrations
+
+### Test your data source connections
+
+Define integrations through the Chalk dashboard and check that they're connecting properly using the `Test Data Source` button.
+![Test Data Source](/img/test-datasource.png)
+
+### Avoid naming your data sources by their "type", e.g. don't call your postgres data source "postgres"
+
+Though this works, it can lead to ambiguity in SQL resolvers when you add multiple data sources of the same type. This occurs
+because [Chalk lets you refer to "single copy" data sources by either their type or their name](/fraud-3#online-data).
```

Comment:
we should link to `fraud-tutorial` instead of the `fraud-{n}` maybe? Also I'm not sure if "single copy" is super standardized, so we might want to explain what that means